src/axom/quest/Shaper.cpp

```diff
@@ -151,23 +216,15 @@ void Shaper::loadShape(const klee::Shape& shape)
   }
 }
 
-void Shaper::applyTransforms(const klee::Shape& shape)
+numerics::Matrix<double> Shaper::getTransforms(const klee::Shape& shape) const
 {
+  numerics::Matrix<double> transformation = numerics::Matrix<double>::identity(4);
```

Comment:
Since this is used in multiple places, I think a constexpr variable (or macro -- horrors!) is better than sprinkling magic numbers in the code.

Reply:
I made a const variable called identity4x4 rather than making multiple calls to the Matrix::identity() function, though that seemed clear...